source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -145,7 +145,11 @@ namespace
             for (const InputPtr& elemInput : nodeElem->getInputs())
             {
                 PvtInput* input = findInputOrThrow(RtToken(elemInput->getName()), node);
-                const string& connectedNodeName = elemInput->getNodeName();
+                string connectedNodeName = elemInput->getNodeName();
```

Comment:
Fixes the node graph connection bug (namely for nodegraphs nodegraph must be specified instead of nodename).

Reply:
So yet another inconsistency with connection handling in core... Good thing we have the runtime model to hide this from the node editor ;) For longer term we should propose a single way to specify connections in core as well.